svixhq/svix-libs
# ruby: initial verification lib

I'm not very familiar with ruby myself but I've added rspec tests which should cover all our bases.

To test you can run:
```sh
bundle install
bundle exec rspec spec
```
We will probably want to add this to `rubygems.org`

Example Usage:
```ruby
require 'svix'

wh = Svix::Webhook.new("MfKQ9r8GKYqrTwjUPD8ILPZIo2LaLaSw")
payload = '{"test": 2432232314}'
headers = {
  "svix-id" => "msg_p5jXN8AQM9LWM0D4loKWxJek",
  "svix-timestamp" => "1614265330",
  "svix-signature" => "v1,g0hM9SsE+OTPJTGt/tmIKtSyZlE3uFJELVlNIOLJ1OE="
}
json = wh.verify(payload, headers)
```

Closes #2.
